Makefile

```diff
@@ -122,12 +117,10 @@ help: ## Show this help. Example: make help
         check-release \
         coverage \
         dist \
-        format \
         lint \
         mypy \
         release \
         run \
         serve-docs \
         test \
-        test-docs \
-        test-release
```

Comment:
This function no longer exists. Gonna move the `.PHONY` commands up as it seems there are a couple that don't exist.